imgoing-frontend/src/components/NotificationBar.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Text, Image } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import styled from 'styled-components/native';
+import FootnoteTypo from './typography/FootnoteTypo';
+
+interface NotificationBarProps {
+  setIsNotiBarVisible: (value: boolean) => void;
+}
+
+const NotificationBarWrapper = styled.View`
+  flex-direction: row;
+  width: 100%;
+  height: 46px;
+  align-items: center;
+  background-color: #f3f6ff;
+  padding-left: 16px;
+`;
+
+const ShortcutButton = styled.Text`
+  text-decoration-line: underline;
+  color: ${({ theme }) => theme.colors.grayHeavy};
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 17px;
+  padding: 0 52px 0 8px;
+`;
+
+const CloseButton = styled.Image`
+  margin-left: 52px;
+`;
+
+const NotificationBar = (props: NotificationBarProps) => {
+  const { setIsNotiBarVisible } = props;
+  return (
+    <NotificationBarWrapper>
+      <FootnoteTypo lang={'ko'} weight={'B'} color={'blue'}>
+        🗣 반복 루틴은 설정에서 관리할 수 있어요.
```

Comment:
```suggestion
        🗣 ${'반복 루틴은 설정에서 관리할 수 있어요.'}
```

이 문구는 알림에 따라서 바뀔 수 있으니 string 관리하는 상수로 빼서 넣는 게 좋을 것같습니다!

Reply:
아예 컴포넌트를 작성해도 좋을 것 같네요. 앞에 있는 이모지도 바뀔 수 있을 것 같습니다!

Reply:
이 컴포넌트는 은성님 말대로 텍스트나 이 모자, 컬러 등을 바꿀 수 있게 작성하는 것이 좋을 것 같아요!